packages/pn-personafisica-webapp/src/components/Contacts/__test__/LegalContacts.test.tsx

```diff
@@ -297,32 +292,4 @@ describe('LegalContactsList Component', async () => {
       expect(result.container).not.toHaveTextContent(defaultAddress!.value);
     });
   });
-
-  it('checks invalid pec', async () => {
```

Comment:
add the test "renders component - no contacts", copied from InsertLegalContact.test.tsx - renders InsertLegalContact